proposals/low-level-struct-improvements.md

```diff
@@ -1461,7 +1470,42 @@ S Usage()
 }
 ```
 
-To make these APIs usable the compiler has to create separation between how freely `ref` and their values can be returned by default. This is the rational for having `ref` to `ref struct` and `out` be [implicitly scoped](#implicitly-scoped). It is the balance that allows APIs like this to be usable.
+To make these APIs usable the compiler has to create separation between how freely `ref` and their values can be returned by default. This is the rational for having `ref` to `ref struct` and `out` be [implicitly scoped](#implicitly-scoped).
+
+It is also why `[UnscopedRef]` only promotes the *ref-safe-to-escape* of such values to *return only* and not *calling method*. Consider that using *calling method* forces a viral use of `[UnscopedRef]` for a `ref struct`:
+
+```c#
+ref struct S
+{
+    byte Field;
+
+    [UnscopedRef]
+    public Span<byte> Data => new Span<byte>(ref Field, 1);
+}
+
+void M(ref S s)
+{
+    // Error: passing a scoped ref to [UnscopedRef] ref 
+    Span<byte> span = s.Data;
+}
+```
+
+This is correctly illegal in that case because the compiler has to consider the pathological case that `S.Data` could cyclic assign via `this`. That forces methods all methods that call `S.Data` to further mark their `ref` parameters as `[UnscopedRef]`. This is viral until the method which creates the value as a local. This is why *return only* exists as an escape scope. It does complicate the spec / implementation but it serves to make the feature significantly more usable.
+
+In terms of advanced annotations the `[UnscopedRef]` design creates the following:
+
+```
+ref struct S { }
+
+// C# code
+S Create1(ref S p)
+S Create2([UnscopedRef] ref S p)
+
+// Annotation equivalent
+scoped<'b> S Create1(scoped<'a> ref scoped<'b> S)
```

Comment:
nit: This notation seems useful, but I don't think we've used it elsewhere in the document. Also the meaning of `>=` isn't self-evident

Reply:
It's used in the advanced annotation section of the spec